reference/v8.json

```diff
@@ -607,6 +608,11 @@
           "js": "0.10.0",
           "ios": "2.0.0",
           "android": "2.0.1"
+        },
+        "auto": {
+          "js": "TBD",
```

Comment:
I assume "TBD" stands for "to be decided." Why "TBD" instead of leaving this `undefined` to indicate that the feature is not supported?